en-US/other/2018/mozfest_tickets.lang

```diff
@@ -0,0 +1,90 @@
+## NOTE: Tweets and email to spread the word about MozFest tickets availability!
+
+
+## MAX_LENGTH: 294
+# Tweet, limit character length to 294. Please rephrase as needed to fit.
+; #mozfest tickets are now on sale! Join us at the world's leading festival for the open internet movement: https://mozillafestival.org/tickets
```

Comment:
@flodolo I’ve added a leading whitespace in English because of the hash, it seems it got removed in other locales. Should I just remove it? I can’t remember if we fixed the issue or not, do you remember?

Reply:
We fixed it for translations https://github.com/mozilla-l10n/langchecker/pull/642